tools/python/convert_from_tensorflow.py

```diff
@@ -417,8 +474,15 @@ def dump_layers_to_file(self, f):
                 if node.op == 'MatMul':
                     self.dump_dense_to_file(node, f)
                 continue
-
-
+            if self.in_batchnorm_scope(node.name):
+                if node.op in self.batchnorm2code:
+                    self.dump_batchnormalization_to_file(node,f)
+                continue
+            elif node.name in self.output_names:
```

Comment:
why need this elif?  all the BN ops should be handled within line 479.

Reply:
Because when the BN operation is the last output, the name node.name of the last child node of the BN operation will become dnn_out, here is to solve this situation.

Reply:
you need a new way to handle this situation, conv2d/dense already shows the method.